modules/custom/az_news/az_news_feeds/src/Form/AzNewsFeedsAdminForm.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+namespace Drupal\az_news_feeds\Form;
+
+use Drupal\Core\Config\ConfigFactoryInterface;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Link;
+use Drupal\Core\Url;
+use Drupal\migrate\Plugin\MigrationPluginManager;
+use GuzzleHttp\ClientInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Form for editing news importer source.
+ */
+class AzNewsFeedsAdminForm extends ConfigFormBase {
+
+  /**
+   * @var \Drupal\migrate\Plugin\MigrationPluginManager
+   */
+  protected $migrationPluginManager;
```

Comment:
Is the `$migrationPluginManager` service actually used in this form anywhere?  Doesn't look like it's needed.